internal/database/database.go

```diff
@@ -178,6 +189,7 @@ func NewDBClient(ctx context.Context, database *azcosmos.DatabaseClient) (DBClie
 	// NewContainer only fails if the container ID argument is
 	// empty, so we can safely disregard the error return value.
 	resources, _ := database.NewContainer(resourcesContainer)
+	billing, _ := database.NewContainer(billingContainer)
```

Comment:
This method already returns an error, being crafty and assuming the ways in which this fails (in the current implementation, not necessarily in the future) IMO just seems like a bad idea. Very simple to check the error and return.

Reply:
That's fair.  Added a new commit to fix this.